gauntlet.sh

```diff
@@ -1,15 +1,17 @@
+#!/bin/bash
+
 # Integration test for copy on demand
 # Tests:
 #  * Full read/write
 #  * Read/write blocks offset by one byte to test for off-by-one errors
 #  * Read/write random sizes and offsets $fuzz_count times
 
 block_size=4096 # 4k bytes
-create_blocks=100 # How many $block_size blocks to generate for test
+create_blocks=262144 # How many $block_size blocks to generate for test
 source_file='./test.dat'
 backing_file='./test.dat.bak'
-cod_executable='/usr/bin/copy-on-demand'
-nbd_device='/dev/nbd0'
+cod_executable='/home/behrlich/copyondemand/copy-on-demand'
+nbd_device='/dev/dblockgauntlet'
```

Comment:
Can you add flags to this integration test so we can test both `nbd` and `kmod-dblock` with integration test suite?